### DISCUSSION TITLE
Good price on a (compatible?) display.

### DISCUSSION THREAD

**Discussion by RussNelson** [mathoudebine/turing-smart-screen-python#164] [General]
https://www.aliexpress.us/item/3256804781685880.html

**Comment by mathoudebine** [comment]
Nice find! In a comment i see `Works fine with windows 11 using Samsung SPF digital photo frame drivers. Search AIDA64 forums for AX206 drivers.` which is confirmed by this other link of the same product: https://fr.banggood.com/3_5-Inch-IPS-Monitor-Screen-AX206-LCD-Display-with-RGB-Breathing-Light-with-4-USB-Port-Support-Raspberry-Pi-p-1971125.html
```
1.Is this small LCD monitor compatible with the 2.4-inch AX206 photo frame?
A: The use of aida64 under Windows is completely compatible, and the Lcd4Linux software package (supporting 32-bit and 64-bit) that needs to be recompiled by myself under Linux is also compatible.
```
The screen is supported by Aida64 on Windows, and on Linux there is the lcd4linux library for the AX206

The AX206 support could be added to this software to support this kind of displays
